Update docs to RFC9580 (#306)
ProtonMail/gopenpgp: README.md

myMessage := decrypted.Bytes()
```

> To encrypt with the [latest proposed OpenPGP standard (RFC9580)](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-13.html):

Reviewer: ```suggestion
To encrypt with the [latest OpenPGP standard (RFC 9580)](https://www.rfc-editor.org/rfc/rfc9580.html):
```

```go
import "github.com/ProtonMail/gopenpgp/v3/profile"

> // Use the profile that conforms with the crypto refresh (RFC9580).

Reviewer: ```suggestion
// Use the latest OpenPGP standard (RFC 9580).
```

pgp := crypto.PGPWithProfile(profile.RFC9580())

> // The default crypto refresh profile uses Argon2 for deriving
> // session keys and uses an AEAD for encryption (AES-256, OCB mode).

Reviewer: ```suggestion
// The RFC9580 profile uses Argon2 for protecting encrypted keys and
// messages encrypted using a passphrase, and uses AEAD for encryption
// (AES-256, OCB mode).
```

Reviewer: There's another instance of "RFC9580 crypto refresh" below that I can't comment on that should be just "RFC 9580".
Also, I think we're missing some examples directly below here?

myMessage := decrypted.Bytes()
```

To encrypt with the [latest OpenPGP standard (RFC 9580)](https://www.rfc-editor.org/rfc/rfc9580.html):
```go
import "github.com/ProtonMail/gopenpgp/v3/profile"

// Use the latest OpenPGP standard (RFC 9580).
pgp := crypto.PGPWithProfile(profile.RFC9580())
// The RFC9580 profile uses Argon2 for protecting encrypted keys and
// messages encrypted using a passphrase, and uses AEAD for encryption
// (AES-256, OCB mode).
// Encrypt data with password
...

> // Decrypt data with password

Reviewer: I think we're missing some examples here?
Also, there's another instance of "RFC9580 crypto refresh" below that I can't comment on that should be just "RFC 9580".
